permission-portal-frontend/client/src/components/NavBar.js

```diff
@@ -61,7 +61,10 @@ const NavBarBase = observer((props) => {
         <div className="title">{props.store.user.isAdmin ? ROLES.ADMIN_LABEL : ROLES.NON_ADMIN_LABEL}</div>
       </div>
       <div className="avatar_group avatar_image">
-        <img src={profile} />
+        <img
+          src={props.store.user.imageBlob ? props.store.user.imageBlob : profile}
+          style={{ objectFit: 'cover', display: 'block', margin: 'auto' }}
```

Comment:
can be in sass file?

Reply:
Probably, though I see other styles scattered about this file and others. I'm going to leave this for now, though a useful task might be to refactor the code to conform to some styling paradigm.

Reply:
i think that is a good call. overall how styles are handled in app is fragmented. @fordacious  - when we refactor, we should move these styles in sass , import and do the usual react way